tests/test_workers/test_pkg_managers/test_npm.py

```diff
@@ -341,6 +341,17 @@ def test_get_deps_allowlisted_file_dep():
     assert replacements == []
 
 
+def test_get_deps_worspaces():
```

Comment:
Can we get a couple more tests if you think it worth it :) For example: a test with a few elements in `workspaces` (all of them are part of `package_lock_deps`, not all of them)

Reply:
sure, good idea :)

Reply:
added